### DISCUSSION TITLE
Issue regarding the reverification of my education profile. While searching for my institution on Bings map.

### DISCUSSION THREAD

**Discussion by Someet-Git** [community/community#60326] [GitHub Education]
### Hi everyone,

1- the bing map is not is not updated 
2- it is not showing some colleges but those are not clickable.
3- why is it saying specific credentials are invalid. what is it needed to create a free developer account here.

This is what it says while searching for my college. 
I don't know how to proceed after this.
![image](https://github.com/community/community/assets/32305867/65ff6f8d-bfef-45b7-8932-1b94dc5b906b)

**Comment by samus-aran** [comment]
Hi @Someet-Git 

Apologies for the delayed response from the team, are you still experiencing this error when searching for your school?

**Reply by Someet-Git** [reply]
Yes, actually my college has 2 campus so this situation.

**Reply by Someet-Git** [reply]
hey @samus-aran I am still facing issue while uploading the school id it is not giving any error but it's showing an error of location of my institute because there are two campus and now I am at my home how can I verify the location I have also raised a support ticket #2255325
and its been 2 months of trying
can you please help me in this regard.


![WhatsApp Image 2023-09-21 at 12 24 49](https://github.com/community/community/assets/32305867/1bcf27a1-1406-47d9-8760-82365b8e8c58)
